src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestHelpers/BaseRunner.cs

```diff
@@ -250,6 +322,29 @@ public virtual bool IsError(FormulaValue value)
         {
             return value is ErrorValue;
         }
+
+        // Derived harness may need to override if they have a different precision level. 
+        public virtual bool NumberCompare(double a, double b)
```

Comment:
>NumberCompare [](http://example.com/codeflow?start=28&length=13)

Different backends have different numerical precision and needed a hook to compare.